projects/cashmere/src/lib/input/hc-form-field.component.scss

```diff
@@ -118,7 +118,16 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
 }
 
 .hc-form-field-disabled {
-    cursor: not-allowed;
+    .hc-form-field-flex {
+        cursor: not-allowed;
+        border: 1.5px solid $gray-200;
+        background-color: $slate-gray-100;
+        color: tint($text, 60%);
+    }
+
+    .hc-input {
```

Comment:
styling specific to hc-input needs to be moved to input.scss. form-field is meant to be used for multiple elements

Reply:
good call - change made